.github/workflows/scrape.yml

```diff
@@ -342,10 +297,10 @@ jobs:
       - name: Commit new data
         working-directory: ./scrapers
         run: |
-          # for directory in $(find courses/* -type d -print0 | xargs -0); do
-          #   DIR_BASENAME=$(basename "$directory")
-          #   cp -r catalog/$DIR_BASENAME/* "$directory"
-          # done
+          for directory in $(find courses/* -type d -maxdepth 0 -print0 | xargs -0); do
+            DIR_BASENAME=$(basename "$directory")
+            cp -r catalog/$DIR_BASENAME/* "$directory" | true
+          done
```

Comment:
Do we want to rsync these directories instead?

Reply:
https://github.com/quacs/quacs/pull/707/commits/b24457a9ca0e7a2f3e9ceb82855ab5378c46c761 Fixed